docs/advanced-topics/reconnecting-mid-game.md

```diff
@@ -0,0 +1,28 @@
+---
+id: reconnecting-mid-game
+title: Reconnecting Mid-Game
+---
+
+In a multiplayer game, it is possible that a client gets disconnected from the server, for example if they have network issues, or if their application or device crashes. When it happens, you might want to give the ability to your players to reconnect to the game.
+
+To allow clients to reconnect in the middle of an active game, there are some considerations to have in mind.
```

Comment:
"The following are some considerations to keep in mind when providing mid-game reconnection capabilities to your players:"